proposal.md

```diff
@@ -1,20 +1,9 @@
-Name: ____________           ID:   ____________
+Name: TraianPop       ID: 43560768
 
 ## Proposed Project
 
-> Replace this paragraph with the description of what your project
-> will do. Tell me what kind of interface it will have, and roughly
-> how it will work. I don't need more than three or four sentences,
-> unless you feel compelled to write more.
-
+I plan on creating a mini-casino simulator that includes some of the more popular gambling games, such as Poker, Blackjack, and Roulette. The interface would involve a user choosing between those three (or more) options and then simply playing against the computer. Alongside this functionality, it should also tell you the probability of winning the hand depending on what you choose, and a check afterwards that makes sure if that's the move you want to do. This project will be to illustrate how unfair most gambling games are, and will not have bluffing as a part of the game.
 ## Outline Structure
 
-> Describe how you'll organize your code. What is the process and
-> supervision structure? If it uses a framework, how does it fit in. I
-> just need to understand the direction you plan to take. Again, three
-> or four sentences are probably enough.
-
-
+My project will first of all have a human interface for the player to interact with the "casino". Then, there will be two modules: one in charge of the deck/numbers and the other for calculating all the percentages involved in the player's decision. Lastly, there will be a supervised server in charge of all the decisions involved in this process. 
```

Comment:
I'm not clear what the supervisor is managing here. What servers will be running.

Reply:
I just realized I didn't really explain a part of my project very well: one of my goals is to have a supervised "casino" server where multiple people can join in and play games, similar to how we did Assignment 4. The children of the casino supervisor would manage all the separate games/tables.